src/pages/event.js

```diff
@@ -127,6 +143,24 @@ export function Event() {
     }
   }, [ensNames]) /* eslint-disable-line react-hooks/exhaustive-deps */
 
+  useEffect(() => {
+    if (loadingEvent === 'succeeded' && csvDownloadIsOnLastStep()) {
+      setCanDownloadCsv(CSV_STATUS.ReadyWithoutEns)
```

Comment:
Can you move this logic from line 148 to 160, to a function called validationDownloadCSV or some related name?